test/expression_test.exs

```diff
@@ -49,6 +49,14 @@ defmodule ExpressionTest do
       assert nil == Expression.evaluate!("@foo[100]", %{"foo" => ["baz", "bar"]})
     end
 
+    test "append lists" do
```

Comment:
Can we add corner cases to the test suite? Like empty lists, nils, etc.

Reply:
I can do it but in the current lib implementation, it will actually throw an error.

Reply:
I think https://github.com/turnhub/expression/pull/151 fixes this issue -- would appreciate a review there :)

Reply:
Approved @smn!
But in the current implementation, [we are ok throwing errors](https://github.com/turnhub/expression/blob/develop/test/expression_test.exs#L332) when manipulating incompatible types such as `nil`, `map`, and `string`. That is something we can revisit later in another PR or lib version.

Reply:
yeah I think we're ok with that. We need to form some good opinions on how to handle errors in expressions anyway, both when they're used as a standalone block `foo > bar` and when they're using in a string `Hello @if(foo > bar, "Anonymous", "Member")`

Reply:
Agree @smn. We would likely change that behaviour in the next `expression` version.